apps/base/namespace-injector-label.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${NAMESPACE}
+  labels:
+    hmcts.github.com/envInjector: enabled
+    slackChannel: ${TEAM_NOTIFICATION_CHANNEL}
```

Comment:
you patch file won't need 
`    slackChannel: ${TEAM_NOTIFICATION_CHANNEL}`


Its added in at the base

Reply:
Removed